domain/src/main/java/com/github/leuludyha/domain/util/FileReader.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.leuludyha.domain.util
+
+import java.io.IOException
+import java.io.InputStreamReader
+
+object FileReader {
```

Comment:
It seems this FileReader is only being used to test data sources, why is it in the domain package ?